_server.js

```diff
@@ -0,0 +1,158 @@
+const electron = require('electron'),
+app = electron.app,
+BrowserWindow = electron.BrowserWindow,
+ipcMain = electron.ipcMain
```

Comment:
Fix lint errors first, so that `npm run lint` passes, then run `npm run clang-format`.

That will get rid of all the nitpicky stuff in reviews about formatting etc.